packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -110,7 +110,7 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 		const uint8ArrayFile = new Uint8Array(file);
 		return this.manager
 			.createBlob(Uint8ArrayToString(uint8ArrayFile, "base64"), "base64")
-			.then((r) => ({ id: r.sha, url: r.url }));
+			.then((r) => ({ id: r.sha, url: r.url, minTTLInSeconds: 86700 }));
```

Comment:
can you make this a constant, and of the format `const minTTLInSeconds = 24 * 60 *60`, also comment that this matches the odsp timestamp. just easier to read and understand where this number comes from